repos/system_upgrade/common/libraries/repofileutils.py

```diff
@@ -38,7 +47,10 @@ def parse_repofile(repofile):
     with open(repofile, mode='r') as fp:
         cp = utils.parse_config(fp, strict=False)
         for repoid in cp.sections():
-            data.append(_parse_repository(repoid, dict(cp.items(repoid))))
+            try:
+                data.append(_parse_repository(repoid, dict(cp.items(repoid))))
+            except Exception as e:
```

Comment:
@tomasfratrik  this is too generic and usually it's not good practice. Specify expected exceptions instead. We know about `ModelViolationError`. Add anything else what you found that could really happen. So far, we know from the history just about this one.

Reply:
Sorry forgot to reply, from testing, only ModelViolationError was present